test/jdk/java/net/InetAddress/getOriginalHostName.java

```diff
@@ -38,19 +40,27 @@ public class getOriginalHostName {
     private static final JavaNetInetAddressAccess jna =
         SharedSecrets.getJavaNetInetAddressAccess();
 
-    public static void main(String[] args) throws Exception {
-        final String HOST = "dummyserver.java.net";
-        InetAddress ia = null;
-        ia = InetAddress.getByName(HOST);
-        testInetAddress(ia, HOST);
-        ia = InetAddress.getByName("255.255.255.0");
-        testInetAddress(ia, null);
-        ia = InetAddress.getByAddress(new byte[]{1,1,1,1});
-        testInetAddress(ia, null);
-        ia = InetAddress.getLocalHost();
-        testInetAddress(ia, ia.getHostName());
-        ia = InetAddress.getLoopbackAddress();
-        testInetAddress(ia, ia.getHostName());
+    public static void main(String[] args) {
+        try {
+            final String HOST = "dummyserver.java.net";
+            InetAddress ia = null;
+            ia = InetAddress.getByName(HOST);
```

Comment:
The try-catch should be around this call only. Stg like:

```
try {
     ia = InetAddress.getByName(HOST);
} catch (UnknownHostException uhe) {
    System.out.println("Skipping " + HOST + " due to " + uhe;
}
testInetAddress(ia, HOST);
```

Reply:
Only catch UnknownHostException for InetAddress.getByName(String). PR has been updated.